components/FollowButton.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <button
```

Comment:
Why not use `BrandedButton`, buttons doesn't have the same size between favorite, modify and see the ruese

Reply:
I just find out that we have a `iconAttrs` props to handle this case ! I'm changing it